src/c++/library/grpc_client.cc

```diff
@@ -76,7 +76,15 @@ GetChannelStub(
 {
   std::lock_guard<std::mutex> lock(grpc_channel_stub_map_mtx_);
 
-  const auto& channel_itr = grpc_channel_stub_map_.find(url);
+  // Specify the number of channels that connect to the url, distribute
+  // clients to different channels relieve the pressure of reaching
+  // max connection concurrency
+  // https://docs.microsoft.com/en-us/aspnet/core/grpc/performance?view=aspnetcore-6.0#connection-concurrency
+  size_t max_share_count = 6;
```

Comment:
const?
Does this need to be a parameter?
Is there an equivalent issue in the python library?

Reply:
This is a better location to link.. https://grpc.io/docs/guides/performance/
See the fourth point. They provide a side note saying:

> Side note: The gRPC team has plans to add a feature to fix these performance issues (see grpc/grpc#21386 for more info), so any solution involving creating multiple channels is a temporary workaround that should eventually not be needed.

Reply:
For Python, we don't attempt channel reuse ourselves, we initialize new grpc channel object for new client. But we also don't do anything to prevent channel reuse within GRPC, assuming Python GRPC has similar behavior as in C++.